xo-install.sh

```diff
@@ -149,13 +149,29 @@ function InstallDependenciesDebian {
 
 	# Install necessary dependencies for XO build
 
-	if [[ $OSVERSION =~ (16|18) ]]; then
+	if [[ $OSVERSION =~ (16|18|20) ]]; then
 		echo -ne "${PROGRESS} OS Ubuntu so making sure universe repository is enabled"
 		add-apt-repository universe >/dev/null
 		echo -e "\r${OK} OS Ubuntu so making sure universe repository is enabled"
 		echo
 	fi
 
+	if [[ $OSVERSION =~ (16|18) ]]; then
+                echo -ne "${PROGRESS} Installing Python for Ubuntu 16 and 18"
+                apt-get install -y  python-minimal >/dev/null
+		echo -e "\r${OK} Python installed"
+                echo
+        fi
+
+	if [[ $OSVERSION =~ (20) ]]; then
```

Comment:
```suggestion
	if [[ $OSVERSION == "20" ]]; then
```
no need to do regex search since it should match only single OS version